metasynth/distribution/legacy/__init__.py

```diff
@@ -0,0 +1,5 @@
+from metasynth.distribution.legacy.regex import RegexDistribution as Regex1_0
+from metasynth.distribution.legacy.regex import UniqueRegexDistribution as UniqueRegex1_0
+
```

Comment:
Maybe keep the Distribution part in the name?

Reply:
I abbreviated it because it was reaching the character limit for this line. I can reinstate it..

Reply:
I think it's better to reinstate and then indent the import line differently. We want the ui for using this distribution to be as similar as possible to the original